mail-filter/rspamd/rspamd-9999.ebuild

```diff
@@ -61,6 +64,11 @@ src_install() {
 	cmake-utils_src_install
 	newinitd "${FILESDIR}/rspamd.init-r2" rspamd
 
+	# Remove mprotect for JIT support
+	if use jit; then
+		pax-mark m "${ED}"/usr/bin/rspamd-* "${ED}"/usr/bin/rspamadm-*
```

Comment:
Can you help me pointing to the right location, where you miss the || die exactly?